backend.tf

```diff
@@ -0,0 +1,48 @@
+terraform {
+  backend "s3" {
+    bucket         = "talent-academy-pascal-lab-tfstates"
+    key            = "talent-academy/vpc_group3/terraform.tfstates"
+    region         = "eu-west-1"
+    dynamodb_table = "terraform-lock"
+  }
+}
+
+# resource "aws_iam_openid_connect_provider" "default" {
```

Comment:
These commented lines should be removed completely.

Reply:
This is done.